doc/GuideForCommitters.md

```diff
@@ -0,0 +1,106 @@
+`openj9-dev@eclipse.org` mailing list after the pull request is merged.
+
+* Do not merge a pull request if you are not one of the Assignees.
+
+* If you will be one of the primary committers, add yourself to the set of
+Assignees. Being the primary committer does not necessarily mean you have to
+review the pull request in detail. You may delegate a deeper technical review
+to a developer with expertise in a particular area by `@mentioning` them. Even
+if you delegate a technical review you should look through the requested changes
+anyway as you will ultimately be responsible for merging them.
+
+* Do not merge a pull request if its title is prefixed with `WIP:` (indicating
+it's a work-in-progress).
+
+* If a review or feedback was requested by the author by `@mentioning` particular
+developers in a pull request, do not merge until all have provided the input
+requested.
+
+* Do not merge a pull request until all validation checks and continuous
+integration builds have passed. If there are circumstances where you believe
+this rule should be violated (for example, a known test failure unrelated to
+this change) then the justification must be documented in the pull request
+prior to merging.
+
+* If a pull request is to revert a previous commit, ensure there is a reasonable
+explanation in the pull request from the author of the rationale along with a
+description of the problem if the commit is not reverted. Additionally, committers 
+should open an issue that links the revert commit to the original PR and tags the 
+original author to ensure that the cause of the revert is addressed and not lost.
+
+* Regardless of the simplicity of the pull request, explicitly Approve the
+changes in the pull request.
+
+
+## Pre-Merge Checklist
+
+* Ensure the pull request adheres to all the Eclipse OpenJ9 [Contribution Guidelines](https://github.com/eclipse/openj9/blob/master/CONTRIBUTING.md).
+
+* Ensure pull requests and issues are annotated with descriptive metadata by
+attaching GitHub labels. The current set of labels can be found [here](https://github.com/eclipse/openj9/labels).
+
+* Be sure to validate the commit title and message on **each** commit in the PR (not
+just the pull request message) and ensure they describe the contents of the commit.
+
+* Ensure the commits in the pull request are squashed into only as few commits as
+is logically required to represent the changes contributed (in other words, superfluous
+commits are discouraged without justification). This is difficult to quantify
+objectively, but the committer should verify that each commit contains distinct
+changes that should not otherwise be logically squashed with other commits in the
+same pull request.
+
+* When commits are pushed to a pull request, TravisCI builds launch automatically 
+to build the changes on x86 Linux (with and without the linter). However, as this only 
+builds the changes, you must initiate a pull request build on all affected platforms 
```

Comment:
current practice is to select a set of platform jdk combos to cover the affected space to try and reduce load on the farm - should there be a few words to that affect here? Perhaps:
builds the changes, you must initiate pull request builds sufficient to test the change across all affected platforms and jdk levels

I just worry if we go with all platforms we may overload the farm because we don't do all with all jdk levels on all the changes at present...

Reply:
That thought occurred to me too, but it does say 'all *affected* platforms': do you think that qualification is not clear enough?

Reply:
@andrewcraik is the _all affected platforms_ qualification good enough, or do you think we need to be more specific in terms of jdk level as well?

Reply:
hmm when I read _affected_ platforms I thought it meant lang level and arch since both can impact a feature. Perhaps a slight loosening like _sufficient to cover affected architectures and language levels_ or something like that?

Reply:
Done.